.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This file should be removed, we don't need editor-specific files in the repo.

Reply:
removed